requirements/requirements.txt

```diff
@@ -11,6 +11,5 @@ tensorboard
 text-unidecode
 torch
 tqdm>=4.41.0
-triton>=3.1.0; sys_platform == 'linux'
```

Comment:
My suggestion: 
- add "arm" platform marker
- separate this dependency
- ignore in `Dockerfile`
This will solve the issue and at the same time will not affect user experience.